pkg/cert/store.go

```diff
@@ -45,38 +45,56 @@ func NewSecretCertStore(namespace, secretName string, clientset clientset.Interf
 	}
 }
 
+// Will attempt to read a secret from the Kubernetes Secret store, format and return it.
+// Will error out if the secret isn't set, or the certificate inside is not properly formatted.
 func (s *secretCertStore) Current() (*tls.Certificate, error) {
+	// Try to get a Certificate from Kube Secret.
+	// This will be there if there's been a successful deployment of the webhook before in this cluster.
 	secret, err := s.clientset.CoreV1().Secrets(s.namespace).Get(
 		s.secretName,
 		metav1.GetOptions{},
 	)
+	// Create an appropriate custom Error for if the secret doesn't exist.
 	noKeyErr := certificate.NoCertKeyError(
-		fmt.Sprintf("no cert/key files read at secret %s/%s",
+		fmt.Sprintf("Error: No data found for cert/key files at secret %s/%s",
```

Comment:
personally I would make it consistent to NOT prefix each error with "Error", that should be evident from the log level (klog.Error). But, I dont really care that much so long as it's consistent.